server.js

```diff
@@ -6038,6 +6039,8 @@ function versionColor(version) {
   var first = version[0];
   if (first === 'v') {
     first = version[1];
+  } else if (/^version\/[0-9]/.test(version)) {
+    version = 'v' + version.substring(8);
```

Comment:
I looked a bit more closely to the code and it looks like I've mistakenly thought the function `versionColor()` was used for the `github/release` integration, but that doesn't seem to be the case. Since you pointed out that this is a rare case, I suggest not to pursue this further.